docs/LabelsNaming.md

```diff
@@ -0,0 +1,83 @@
+# Test Labels
+
+We would like to also have tests distinguished by labels, not only by folders for specific components. We can also run the tests only with specific label(s).
+
+## Usage
+
+See: https://onsi.github.io/ginkgo/#spec-labels 
+
+## Types of labels
+- component
+- test type
+- category
+
+### Component Labels
+
+Component Labels | Description
+--- | --- 
+has | HAS related tests
+build | Build related tests
+cluster-registration | cluster registration related tests
+e2e-demos | e2e-demos related tests
+
+### Test Types Labels
+
+Test Types Labels | Description
+--- | --- 
+slow | Slow tests (See --slow-spec-threshold (https://onsi.github.io/ginkgo/#other-settings))
+e2e-demos | e2e demos tests
+load | Load tests
+performance | Performance related tests
+smoke | Critical functionality tests
+serial | Tests which can’t be run in parallel
+security | Security related tests
+
+### Test Stability Labels
+
+Test Types Labels | Description
+--- | --- 
+flaky | Flaky tests (all tests are stable unless marked flaky)
+
+### Test Categorization Labels
+
+Test Types Labels | Description
+--- | --- 
+customer-feedback | Test created upon feedback from any customer channel (customer issue, telemetry data, …)
+demo | Tests related to milestone demos
+
+# Tests naming 
+
+Tests should be named like this:
+```
+...
+Describe("[test_id:01][crit:high][posneg:negative]should work", Labels("has", "slow"), func() {
```

Comment:
Do I get it correctly, that the `test_id` is mandatory? (And that it's the only mandatory field?) If yes, can it be explicitely written here?
What happens if some test case doesn't have `test_id` setup?

Reply:
I have decided to remote the test_id right now from this PR(I will leave it only in docs and HAS suites), I will create another PR with test_ids for the rest and updated documentation.